__visual_tests__/utils.ts

```diff
@@ -0,0 +1,36 @@
+import * as os from 'os';
+
+import {expect, Page} from "@playwright/test";
+
+import {WidgetComponentIds} from "./const";
+
+export async function assertScreenshot(page: Page, screenshotName: string, browserName: string) {
+  const arch = os.arch();
+  const name = `${screenshotName}.${browserName}.${arch}.png`; // Include the browser and OS architecture info in the filename
```

Comment:
![image](https://github.com/user-attachments/assets/827abd4b-2791-4e85-ab5c-5fd753ba3b18)
playwright 에서는 process.platform 사용한다고 하네요
```suggestion
import {expect, Page} from "@playwright/test";

import {WidgetComponentIds} from "./const";

export async function assertScreenshot(page: Page, screenshotName: string, browserName: string) {
  const name = `${screenshotName}.${browserName}.${process.platform}.png`; // Include the browser and OS architecture info in the filename
```